src/test/kotlin/Utilities/FileIoTest.kt

```diff
@@ -0,0 +1,22 @@
+import Utilities.FileIo
+import kotlin.test.*
+
+class FileIoTest {
```

Comment:
might not hurt to test some edge cases:
- empty file
- no file (can be that it just throws an error)

Reply:
Seconded.